js/fairdesk.js

```diff
@@ -0,0 +1,1450 @@
+        // {
+        //     "status": "0",
+        //     "error": "OK",
+        //     "data": [
+        //       {
+        //         "symbol": "btcusdt",
+        //         "positionSide": "LONG",
+        //         "isolated": false,
+        //         "avgEntryPrice": "35907.49953800",
+        //         "quantity": "0.01300000",
+        //         "markPrice": "34112.0",
+        //         "isolatedMargin": "0.00000000",
+        //         "frozenMargin": "0.00000000",
+        //         "margin": "",
+        //         "liqPrice": "",
+        //         "lastFundingTime": "0",
+        //         "lastTxTime": "1651907743728",
+        //         "baseCurrency": "BTC",
+        //         "quoteCurrency": "USDT",
+        //         "settleCurrency": "USDT",
+        //         "curTermRealisedPnl": "0.00000000",
+        //         "curTermFirstOrderId": "45844695358",
+        //         "cumRealizedPnl": "-9628.25998900",
+        //         "arenaTrade": null
+        //       }
+        //     ]
+        // }
+        const positions = this.safeValue (response, 'data', {});
+        const result = [];
+        for (let i = 0; i < positions.length; i++) {
+            const position = positions[i];
+            result.push (this.parsePosition (position));
+        }
+        return this.filterByArray (result, 'symbol', symbols, false);
+    }
+
+    parsePosition (position, market = undefined) {
+        const marketId = this.safeString (position, 'symbol');
+        market = this.safeMarket (marketId, market);
+        const symbol = market['symbol'];
+        const margin = this.safeString (position, 'margin');
+        const positionSide = this.safeStringLower (position, 'positionSide');
+        const marginType = this.safeString (position, 'isolated') === 'true' ? 'isolated' : 'cross';
+        const timestamp = this.safeInteger (position, 'lastTxTime');
+        const qty = this.parseNumber (this.safeString (position, 'quantity'));
+        const entryPrice = this.parseNumber (this.safeString (position, 'avgEntryPrice'));
+        const markPrice = this.parseNumber (this.safeString (position, 'markPrice'));
+        const notional = qty * markPrice;
+        const sign = positionSide === 'long' ? 1 : -1;
+        const unrealizedPnl = (markPrice - entryPrice) * qty * sign;
+        return {
+            'info': position,
+            'symbol': symbol,
+            'positionSide': positionSide,
+            'quantity': qty,
+            'unrealizedPnl': unrealizedPnl,
+            'margin': this.parseNumber (margin),
+            'notional': notional,
+            'markPrice': this.parseNumber (this.safeString (position, 'markPrice')),
+            'entryPrice': entryPrice,
+            'timestamp': timestamp,
+            'initialMargin': marginType === 'isolated' ? this.safeString (position, 'isolatedMargin') : undefined,
```

Comment:
We must follow this rule: https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#using-ternary-conditionals